ledController/src/commandhandler.py

```diff
@@ -24,7 +24,7 @@ def _create_error_msg(self, msg):
         return result
     
     def initial_message(self):
-        return {"client": "raspberry"}
+        return {"client": "ledController"}
```

Comment:
If you change to snake case in the other PR, do so here as well. If not, ignore this comment.